_articles/en-US/how-to-contribute.md

```diff
@@ -45,7 +45,7 @@ Working with others on a shared project means you'll have to explain how you do
 
 ### Build public artifacts that help you grow a reputation (and a career)
 
-By definition, all of your open source work is public, which means you get free examples to take anywhere as a demonstration of what you can do.
+By definition all of your open source work is public; which means you get free examples to take anywhere as a demonstration of what you can do.
```

Comment:
The sentence is a bit unwieldy but I'm not sure this is the right fix. Maybe there should be a period after public and which should be This?

Reply:
I like @mlinksva's solution better